src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img src="/moyo_1.svg" alt="Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">APPLE</a></li>
```

Comment:
1. apply code styling recommendation from a [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) all over the page
2. use lowercased values at html and change it at css
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              apple
            </a>
          </li>
```